test/functional/Jsr292/playlist.xml

```diff
@@ -137,6 +137,40 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>jsr292Test_SM</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+			<variation>Mode195</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JAVA_SECURITY_MANAGER) $(JVM_OPTIONS) \
+	--add-opens=java.base/java.lang=ALL-UNNAMED \
+	-Djava.security.policy=$(Q)$(TEST_RESROOT)$(D)java.policy$(Q) \
+	-cp $(Q)$(TEST_RESROOT)$(D)jsr292test.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng_sm.xml$(Q) \
+	-testnames jsr292Test_SM \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<versions>
+			<version>11</version>
+			<version>17</version>
+			<version>18</version>
+		</versions>
+		<!--
+			TODO: Further investigation is needed for running against non-openj9 sdk
+		-->
+		<impls>
+			<impl>openj9</impl>
+			<impl>ibm</impl>
+		</impls>
+	</test>
```

Comment:
Instead of add `-testnames jsr292Test_SM` in different test targets , could we just have one `jsr292Test_SM` target? And add `<version>8</version>` and `<variation>-Xjit:count=0</variation>` in this `jsr292Test_SM` target? Did I miss anything?

Reply:
It seems like `jsr292Test_jdk8` and `jsr292Test_jdk8_special` are the same (with variation changes) but `jsr292Test_JitCount0_jdk8` has an extra testname `jsr292Test_optional` and platform requirement `^arch.arm`. The first two could probably be merged but why are they separate to begin with? If I merge the new SM tests then the non SM tests should be merged as well.

Reply:
> The first two could probably be merged but why are they separate to begin with? 

I am not sure the details, but I think these tests were added in the early days. And the test framework (TKG) might have limited ability back then, which resulted in separated targets. 

> If I merge the new SM tests then the non SM tests should be merged as well.

I was thinking to keep this PR simple and only deal with SM tests. And we can have a separate PR for cleaning up non SM tests later. But it is up to you. I am ok either way.

Reply:
The github diff was messy when I tried to merge everything so I merged just the SM tests instead